packages/introspect/src/generate.ts

```diff
@@ -177,10 +189,17 @@ function createTableJsonSchema(table: TableDetails): JSONSchema {
 function createRootJsonSchema(
     schemas: Record<string, Schema>,
     { defaultSchema }: { defaultSchema: string },
```

Comment:
don't add a third parameter, instead add "incldueTables" to the second "props-object" that currently only contains defaultSchema.